pkgs/by-name/ra/radiance/package.nix

```
@@ -0,0 +1,34 @@
+{
+  lib,
+  stdenv,
+  cmake,
+  libGL,
+  libGLU,
+  fetchFromGitHub,
```

From what I've seen there is a preference to format arguments as follows:

```suggestion
{ lib
, stdenv
, cmake
, libGL
, libGLU
, fetchFromGitHub
```